android/app/build.gradle

```diff
@@ -22,7 +22,7 @@ buildscript {
       googleFirebaseServicesDefault
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:8.3.2'
+    classpath 'com.android.tools.build:gradle:8.4.0'
```

Comment:
I'd rather wait until 8.4.1 is out.. There is no need to rush, its better to wait until newly introduced bugs are found and fixed.

Also 8.4 introduces new minification changes which should be tested thoroughly https://developer.android.com/build/releases/gradle-plugin#library-classes-shrunk

Also devs will have to upgrade Android Studio to `Jellyfish | 2023.3.1`

Reply:
Minifying apk and upgrading to the latest software are good things, no?

@Jean-BaptisteC does it work on all your devices? @rtsisyk can also run it on monkeys if necessary, but IMO it would be naturally tested in master, and then in betas. In case of any issues found the commit can be easily reverted.

Reply:
Postponing issue discovery for later is also not a good long-term strategy. Don't forget about pros of the upgrade: https://developer.android.com/studio/releases/fixed-bugs/studio/2023.3.1

Reply:
@rtsisyk can you please check and help with fixing the linter error?

Reply:
@rtsisyk bump